### DISCUSSION TITLE
Updating the SSL certificate

### DISCUSSION THREAD

**Discussion by egasvegas1109** [XTLS/Xray-core#3694] [Q&A]
Hello, I am using a certificate from Let's Encrypt, which is updated using cerbot. What happens when the certificate is updated? Will xray pull it up automatically or do I have to restart it?

**Comment by meower1** [comment]
there won't be any issues after certificate renewal. it will remain the same. you don't have to touch anything

**Reply by egasvegas1109** [reply]
what does "it will remain the same" mean? cerbot also issues a new serificate once every 90 days (updates the old one)

**Reply by meower1** [reply]
as in the location of the certificate will remain the same so no changes would be needed
